project/Build.scala

```diff
@@ -864,6 +864,9 @@ object Build {
         } (Set(scalaLibrarySourcesJar)).toSeq
       }.taskValue,
       sources in Compile ~= (_.filterNot(file =>
+        // Ported filles from https://github.com/scala/scala/pull/9307
+        // TODO: Remove this and `stdlib-bootstrapped/src` files when the library is updated above 2.13.3
```

Comment:
This can be removed now that https://github.com/lampepfl/dotty/pull/10392 has been merged.